Tests/README.md

```diff
@@ -21,33 +21,50 @@ TOOLS_PATH = folder where the tools are located (e.g. reference decoder, MCTS de
 These folders should be created prior to running the script.\
 The bitstream folder should be empty and cleared for each run.\
 Note: You should build the SVT encoder and place the executable in the folder specified under "ENC_PATH"\
-Note: Download the reference decoder from https://hevc.hhi.fraunhofer.de/ and place in the folder specified under "TOOLS_PATH"\		
+Note: Download the reference decoder from https://hevc.hhi.fraunhofer.de/ and place in the folder specified under "TOOLS_PATH"\
```

Comment:
```suggestion
Note: Download the reference decoder from https://hevc.hhi.fraunhofer.de/ and place in the folder specified under "TOOLS_PATH"
```
This one doesn't need `\` at the end since you have at least one empty line between this and the next

Reply:
addressed..